### ISSUE TITLE
when run the code

### ISSUE THREAD

**Issue by Qudor-Engineer** [spyder-ide/spyder#7709]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "/usr/lib/python3.6/site-packages/spyder/app/mainwindow.py", line 2463, in execute_in_external_console
    console.execute_code(lines)
  File "/usr/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 954, in execute_code
    sw.execute(to_text_string(lines))
  File "/usr/lib/python3.6/site-packages/qtconsole/history_console_widget.py", line 47, in execute
    source, hidden, interactive)
  File "/usr/lib/python3.6/site-packages/qtconsole/console_widget.py", line 631, in execute
    self._execute(source, hidden)
  File "/usr/lib/python3.6/site-packages/qtconsole/frontend_widget.py", line 294, in _execute
    msg_id = self.kernel_client.execute(source, hidden)
AttributeError: 'NoneType' object has no attribute 'execute'

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.5
* Qt Versions: 5.9.4, PyQt5 5.9.1 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.5.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.1 (OK)
numpy >=1.7      :  1.12.1 (OK)
sphinx >=0.6.6   :  1.5.6 (OK)
rope >=0.9.4     :  0.10.7 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
psutil >=0.3     :  5.4.3 (OK)
nbconvert >=4.0  :  4.2.0 (OK)
sympy >=0.7.3    :  None (NOK)
cython >=0.21    :  None (NOK)
qtconsole >=4.2.0:  4.2.1 (OK)
IPython >=4.0    :  5.3.0 (OK)
pylint >=0.25    :  1.7.5 (OK)

```

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.3.1). Please update.

**Comment by Qudor-Engineer** [comment]
@ccordoba12 
thank you so much it solved my problem
